### ISSUE TITLE
GGMax random crashes

### ISSUE THREAD

**Issue by Bullyblasta** [TheGameCreators/GameGuruRepo#3616]
doesnt matter what im doing just randomly crashes, like after i save level, or if i step away to go to bathroom or something. i have nothing custom in the engine.

**Comment by megold16** [comment]
Mine has been crashing too. Every time I open up GG since the last update, it crashes about 15 minutes in, while i'm sculpting terrain. Aslo, the auto updater keeps throwing an error code.

**Comment by Tam51** [comment]
Are you running the non-steam version. If so then it should be fixed on the next update which I think is on Friday this week. As mine is working fine

**Comment by Bullyblasta** [comment]
OK thank you.

**Comment by davetgc** [comment]
Hi, thanks for the report. This is fixed for the Steam version, are you using the TGC one?

**Comment by Bullyblasta** [comment]
yes i got mine from the TGC website.

**Comment by Bullyblasta** [comment]
Yes I got GGMax from TGC website Sent from Mail for Windows From: davetgcSent: Thursday, August 18, 2022 11:12 AMTo: TheGameCreators/GameGuruRepoCc: Bullyblasta; AuthorSubject: Re: [TheGameCreators/GameGuruRepo] GGMax random crashes (Issue #3616) Hi, thanks for the report. This is fixed for the Steam version, are you using the TGC one?—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>
